Makefile

```diff
@@ -29,6 +27,9 @@ else
 endif
 
 
+-include .make_options
```

Comment:
Why this was moved down here?

Reply:
Hmm, at first I thought it seemed nicer, but on second thought it really should be above. I'll move it back.